r/R/csv.R

```diff
@@ -455,25 +457,32 @@ CsvReadOptions$create <- function(use_threads = option_use_threads(),
   options
 }
 
-readr_to_csv_write_options <- function(include_header,
-                                       batch_size = 1024L) {
+readr_to_csv_write_options <- function(include_header = TRUE,
+                                       batch_size = 1024L,
+                                       na = "") {
   assert_that(is_integerish(batch_size, n = 1, finite = TRUE), batch_size > 0)
   assert_that(is.logical(include_header))
+  assert_that(is.character(na))
+  assert_that(length(na) == 1)
+  assert_that(!grepl('"', na), msg = "na argument must not contain quote characters.")
```

Comment:
Should this assertion move down into `CsvWriteOptions$create`?

Reply:
Good point. I don't even know why we have the assertions here if it just calls into `CsvWriteOptions$create`